### PR TITLE
Add a new README section about how to add small edits on the doc

### PR DESCRIPTION
**Pull request by whitphx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ In the different folders, you will find:
 - The tasks as visible on the following page: [hf.co/tasks](https://hf.co/tasks) in the `tasks` folder.
 - In the `docs` folder, the Hugging Face Hub documentation as visible here: [hf.co/docs/hub](https://hf.co/docs/hub)
 
+### How to contribute to the document
```
Comment (osanseviero) [inline README.md:10]:
```suggestion
### How to contribute to the docs

```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ In the different folders, you will find:
...
+Just add/edit the Markdown files, commit them, and create a PR.
+Then the CI bot will build the preview page.
```
Comment (osanseviero) [inline README.md:12]:
```suggestion
Then the CI bot will build the preview page and provide a url for you to look at the result!
```

**Review by osanseviero — APPROVED** [review]
Thanks a lot! Just some minor nits

**Comment by whitphx** [comment]
Thanks for the edits and approval!

**Comment by whitphx** [comment]
Oops I merged this PR and removed the branch before the bot creates the preview... were these CI errors due to it?

**Comment by julien-c** [comment]
no, should be good 👍

**Pull request merged**